commons/signservice-commons/src/main/java/se/idsec/signservice/security/sign/impl/DefaultSignatureValidationResult.java

```diff
@@ -198,6 +208,48 @@ public void setCertificateValidationResult(final CertificateValidationResult cer
     this.certificateValidationResult = certificateValidationResult;
   }
 
+  /** {@inheritDoc} */
+  @Override public String getSignatureAlgorithm() {
+    return signatureAlgorithm;
+  }
+
+  /**
+   * Assigns the signature algorithm URI of the signature
+   * @param signatureAlgorithm
+   *          signature algorithm URI
+   */
+  public void setSignatureAlgorithm(String signatureAlgorithm) {
+    this.signatureAlgorithm = signatureAlgorithm;
+  }
+
+  /** {@inheritDoc} */
```

Comment:
@Override on its own line
this.claimedSigningTime 

Just to be consistent with other code

Reply:
Fixed